### PR TITLE
Format repo and use formatter workflow (2nd try)

### PR DESCRIPTION
**Pull request by JoshuaLampert**
See #86.

### CODE REVIEW

**Comment by mikeingold** [comment]
I just pushed a commit to tweak a few lines in the test section, mostly just to condense a few lines that were being split in two.

File: src/integration_rules.jl
[the remark this answers is not in the document]
Comment (mikeingold) [reply src/integration_rules.jl:1]: I just went ahead and made the change, specifying the fields as `Base.Pairs`

[the remark this answers is not in the document]
Comment (JoshuaLampert) [reply src/integration_rules.jl:1]: I also noticed that. Looks good. Thanks!

File: test/floating_point_types.jl
[the remark this answers is not in the document]
Comment (JoshuaLampert) [reply test/floating_point_types.jl:1]: Yeah, that's annoying and tracked in https://github.com/domluna/JuliaFormatter.jl/issues/737.

[the remark this answers is not in the document]
Comment (mikeingold) [reply test/floating_point_types.jl:1]: If we revert the non-conforming `@test` changes, will the JuliaFormatter bot/Action pitch a fit about it?

[the remark this answers is not in the document]
Comment (JoshuaLampert) [reply test/floating_point_types.jl:1]: Yes, it will fail and will make a comment "fixing" it.

[the remark this answers is not in the document]
Comment (mikeingold) [reply test/floating_point_types.jl:1]: I'm not a fan of the bug causing these particular `@test` whitespace changes, but I think the value provided everywhere else is probably worth it on balance 🤷‍♂️

**Review by mikeingold — APPROVED** [review]

File: test/floating_point_types.jl
[the remark this answers is not in the document]
Comment (JoshuaLampert) [reply test/floating_point_types.jl:1]: Yes, I have the same opinion.

**Pull request merged**